_query-dsl/specialized/percolate.md

```diff
@@ -0,0 +1,704 @@
+---
+layout: default
+title: Percolate
+parent: Specialized queries
+nav_order: 55
+---
+
+# Percolate
+
+Use the `percolate` query to find stored queries that match a given document. This is the opposite of a regular search. Instead of finding documents that match a query, you find queries that match a document. Percolate queries are often used for alerting, notifications, and reverse search use cases.
```

Comment:
```suggestion
Use the `percolate` query to find stored queries that match a given document. This operation is the opposite of a regular search: instead of finding documents that match a query, you find queries that match a document. Percolate queries are often used for alerting, notifications, and reverse search use cases.
```